package.json

```diff
@@ -77,6 +79,10 @@
       },
       "locator": "cordova-plugin-crosswalk-webview",
       "id": "cordova-plugin-crosswalk-webview"
+    },
+    {
+      "locator": "../me.apla.cordova.app-preferences",
```

Comment:
I think that @apla has fixed this now. We should try again, but at least, we should not check in the relative path.